src/main/java/com/kiwigrid/helm/maven/plugin/InitMojo.java

```diff
@@ -135,6 +135,13 @@ protected void downloadAndUnpackHelm() throws MojoExecutionException {
 	}
 
 	private void verifyLocalHelmBinary() throws MojoExecutionException {
+		getLog().info("Run helm init...");
```

Comment:
This duplicates line 128ff.
Pls. extract this to a method.

Reply:
Ok

Reply:
Changes done